### DISCUSSION TITLE
Google Ads Connector no longer performing normalization

### DISCUSSION THREAD

**Discussion by iainad** [airbytehq/airbyte#29098] [Connector Questions]
I've been struggling with some connection issues and done a fresh install of Airbyte but I've noted that my Google Ads connector (0.74) is no longer performing normalizations on the data.

Is this expected behaviour? I note a comment in [another question](https://github.com/airbytehq/airbyte/discussions/28474) about normalization being depreciated.

I can see no way in the UI to turn on normalization again and I set up a new connection from Google Ads to BigQuery which just sent the raw tables.

Ideally if I could restore the normalization for now this would be simpler than creating new normalized views from the raw tables but I guess that's the alternative I'm left with if not.

Am running self hosted 0.50.13 with Big Query destination 1.7.3 so all looks to be fully up to date. Sync is working fine just normalization is being skipped.

Any guidance appreciated!

Thanks

**Comment by marcosmarxm** [comment]
@iainad are you able to see the tab Replication in your connection? Do you have other connections using the Bigquery destination?

**Reply by iainad** [reply]
@marcosmarxm i can see the tab but it doesn't have any detail about normalisation below the list of tables which is where I think it used to be. 

I created a new destination to test in case it was related so I get the same behaviour on the existing destination (but it does have the normalised tables which have not been updated since 18th July when I did the upgrade. The new destination only has the raw tables. 

So am a bit stumped really. I had to migrate settings from a previous install which was failing and I used Octavia-cli to import config. But the new destination I configured manually so don't think it's related to that.

[a comment by an automated account is not shown]
